src/styles/variablesForJS.scss

```diff
@@ -0,0 +1,8 @@
+@import './variables';
+
+:export {
+  breakMobile : $break-mobile;
+  breakTablet : $break-tablet;
+  breakDesktop : $break-desktop;
+  breakDesktopWide : $break-desktop-wide;
+}
```

Comment:
@homeday-de/frontend-engineering : Any potential issue you see here, whit exposing those variables and using them?

Reply:
Looks good to me.

Reply:
I think it is useful, sliders (https://github.com/homeday-de/sliders/blob/develop/src/styles/variables.scss#L101) also uses it.

Reply:
We remembered you've demoed this with sliders - and used it! :)

Reply:
The solution above won't work cross projects, because these variables are meant to be overwritten by each project, to match the needed breakpoints. So `sassVariables.breakTablet` will always be `768px` (as defined [here](https://github.com/homeday-de/homeday-blocks/blob/f0cee4815e333cfd15cba716b9a1354eea7a5a03/src/styles/_variables.scss#L50
)), while in myHomeday for example it should be `600px`.

I suggest using the breakpoints service for this, and defining the breakpoints globally on each project.
Or passing the breakpoint as a prop (ad having `sassVariables.breakTablet` as default).

Reply:
Yeah, that's a concern. @aym3nb can you create an update ticket for this one? So we address it better. We can pass it as prop, would make it more robust and inline with our breakpoints strategy (with default value). :+1:

Reply:
Actually, we declare variables as `default` in our scss - https://github.com/homeday-de/homeday-blocks/blob/develop/src/styles/_variables.scss#L49 
So if they are overwritten in the project, they would be picked up correctly during the build here as well, wouldn't they? Otherwise breakpoints of the project would not be inline with breakpoints here. 
Anyway, I guess my point is, I need to try it, to wrap my head properly around it. :) As this is already used in myHomeday, perhaps we can quickly check it there, if correct value is picked up or not?

Reply:
My guess is that they wouldn't, because we are getting the variables directly from the `_variables.scss` file, and not from where they are overwritten.
Yeah, let's try it and see.

Reply:
Those values would be set only if you did not define them. Have a look here: https://sass-lang.com/documentation/variables#default-values

> To make this possible, Sass provides the !default flag. This assigns a value to a variable only if that variable isn’t defined or its value is null. Otherwise, the existing value will be used.

So I think it could work. I imagine we inject our css variables at beginning with webpack, then the rest is loaded. Agree, it's to abstract for me to be certain at this point, so let's try it. :)

Reply:
Just quickly checked the build of [myHomeday](https://my.homeday.de/js/app.85400a0c.js). 

```
{breakMobile:"460px",breakTablet:"600px",breakDesktop:"900px",breakDesktopWide:"1200p"}
```

Looks good. :ok_hand:

Reply:
I was shocked to see that it works too.

I did some tests and found out that maybe it works because we are still using the auto scss variables import in myHomeday. I assume that line gets injected in the beginning of `variablesForJS.scss` as well.

Snipped from vue.config.js:
```js
module.exports = {
...
css: {
    loaderOptions: {
      sass: {
        prependData: `
          @import "@/styles/variables.scss";
        `,
      },
    },
  },
};
```
Which means that even the `@import './variables';` is not needed in that case and you'll have access to all the projects variables (for example `$sidebar-width` which is myHomeday specific).

**tl;dr**
It will work just for myHomeday and not the other projects.